lib/companies_house/client.rb

```diff
@@ -93,7 +104,8 @@ def request(resource,
                 path,
                 params = {},
                 transaction_id = make_transaction_id,
-                resource_id = nil)
+                resource_id = nil,
+                headers = {})
```

Comment:
I see you've added the new `headers` parameter but don't seem to be using it in the consumer, is this intentional?

Reply:
I use it like this:

```
    def content_object(document)
      content_uri = URI(document['links']['document'])
      content_client = client("#{content_uri.scheme}://#{content_uri.host}")
      content_client.send(
        :request, :document, content_uri.path, {}, SecureRandom.hex(10), nil,
        'Accept': document['resources'].keys.first
      )
    end
```

Reply:
Would it help to make the `#request` method public or maybe add another public method that does this for you?

Reply:
Yeah, either would perhaps work. Not sure what your preferred approach would be on that, but happy to be guided. Definitely a bit of monkeying around going on with my calling private methods with `send()` there.

Reply:
I reckon making this method public would be valuable if you wouldn't mind changing it - then when we don't expose a helper method for the endpoint, you can still use the underlying configured client to save yourself doing something scary.

Happy for you to do this in this PR or a follow-up, it should be a small change!